src/machine/behavior/cu.rs

```diff
@@ -1 +1,25 @@
+use machine::state::State;
+use machine::state::mem::ByteAt;
 
+pub fn cycle(state: &mut State) {
+
+    assert_eq!(state.pc % 4, 0);
+
+    // fetch opcode and operands
```

Comment:
For me, these comments are headings and headings start with a capital letter... 	😉